src/components/Courses/Components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '../../../../common/Button/Button';
+
+type SearchBarProps = {
+  children: React.ReactNode;
+};
+
+export const SearchBar = (props: SearchBarProps) => {
+  return (
+    <div>
+      {props.children}
+      <Button
+        buttonText='SEARCH'
+        onClick={() => console.log('SEARCH button was clicked')}
```

Comment:
It seems you're already filtering when the text changes in the search bar input

Reply:
Fixed

Reply:
Fixed. Now it filters only when the Search button is pressed.